pkg/util/util.go

```diff
@@ -399,3 +401,48 @@ func Zipit(source, target string) error {
 
 	return err
 }
+
+// DocExists checks whether an elasticsearch doc of a testrun exists in index testmachinery-* index
+func DocExists(log logr.Logger, esConfig *elasticsearch.Config, testrunID, testrunStartTime string) (docExists bool) {
+	log.Info("check if docs have already been ingested")
+
+	payload := fmt.Sprintf(`{
+			"size": 0,
+			"query": {
+				"bool": {
+					"must": [
+						{ "match": { "tm.tr.id.keyword": "%s" } },
+						{ "match": { "tm.tr.startTime": "%s" } }
+					]
+				}
+			}
+		}`, testrunID, testrunStartTime)
+
+	if esConfig == nil {
+		log.Info("no elasticsearch config available, cannot check if doc already exists")
+		return false
+	}
+	esClient, err := elasticsearch.NewClient(*esConfig)
+	if err != nil {
+		log.Error(err, "couldn't create elasticsearch client")
+		return false
+	}
```

Comment:
we already create a client in the `ingestDir`, maybe we should only pass the client to have the same code twice.

Reply:
this makes sense after removal of deprecated es.configname approach. Therefore postponed